src/features/build_features.py

```diff
@@ -52,12 +56,20 @@ def __init__(
         self.input_test_data = input_test_data
         self.save_data = save_data
         self.output_file_names = output_file_names
+        self.cols_to_drop = cols_to_drop
+        self.pipeline_dir = pipeline_dir
+        self.pipeline_path = get_data_path() / pipeline_dir
+        self.pipeline_name = f"feature_pipeline_{clusters_location['n_clusters']}_{clusters_trip['n_clusters']}.pkl"
```

Comment:
I cannot accept this way of naming pipeline, due to the fact that not all params are taken into consideration. I propose hashing all params and providing hash   name. 

```
params = str(geodata) + str(clusters_location) + str(clusters_trip) + str(trip_distance_model)
params_md5 = hashlib.md5(params.encode("utf-8")).hexdigest()
self.pipeline_name = f"{params_md5}.pkl"
```